frontend/lib/src/components/widgets/DataFrame/columns/ChartColumn.test.ts

```diff
@@ -96,6 +106,14 @@ describe("ChartColumn", () => {
     expect((mockBarChartCell as SparklineCellType).data?.graphKind).toEqual(
       "bar"
     )
+
+    const mockAreaChartColumn = getAreaChartColumn()
+    expect(mockAreaChartColumn.kind).toEqual("area_chart")
+    const mockAreaChartCell = mockAreaChartColumn.getCell([0.1, 0.2, 0.3])
+    // Chart type should be bar
```

Comment:
nit: I am not sure what the comment is referring to here 🤔

Reply:
Good catch. Copy & paste error. Fixed it 👍